File: notify/sendToListMms.js
// -----------------------------------------------------------------------------
console.log("+++ Start sending notifications to a list.");

const client = require('twilio')(process.env.MASTER_ACCOUNT_SID, process.env.MASTER_AUTH_TOKEN);

const notifyServiceSid = process.env.MASTER_NOTIFY_SERVICE_SID;
const phoneNumber1 = process.env.MY_PHONE_NUMBER;
console.log("+ notifyServiceSid: " + notifyServiceSid + " Send to: " + phoneNumber1);
// Doesn't work to send the image:
client.notify.services(notifyServiceSid).notifications.create({
    body: 'Hello there from Notify 3',
    sms: { 'media_urls':['https://www.tigerfarmpress.com/StacyDavid/netscape-fountain2a.jpg']},
    toBinding: [
        JSON.stringify({
            "binding_type": "sms", 
            "address": phoneNumber1
        })
    ]
}).then(notification => console.log(notification.sid))
        .catch(error => console.log(error));

File: notify/sendToListMms.py
print("+++ Send Notify MMS.")
import os
import json
from twilio.rest import Client
account_sid = os.environ.get("ACCOUNT_SID")
auth_token = os.environ.get("AUTH_TOKEN")
client = Client(account_sid, auth_token)
notify_service_sid = os.environ.get("MASTER_NOTIFY_SERVICE_SID")
phoneNumber1 = os.environ.get("MY_PHONE_NUMBER")
print("+ Using notification service, SID: " + notify_service_sid)
print("+ Send notifications to: " + phoneNumber1)
notification = client.notify.services(notify_service_sid).notifications.create(
    body='Hello to you two 3',
    sms={ 'media_urls':['https://demo.twilio.com/owl.png']},
    to_binding=[
        json.dumps({"binding_type":"sms", "address": phoneNumber1})
    ]
  )
print("+ Notification SID: " + notification.sid)
print("+++ Exit.")
# ------------------------------------------------------------------------------
